frb_codegen/src/commands.rs

```diff
@@ -245,3 +250,82 @@ pub fn build_runner(dart_root: &str) -> Result {
     }
     Ok(())
 }
+
+lazy_static! {
+    static ref CACHE: Mutex<HashMap<String, String>> = Mutex::new(HashMap::new());
+}
+
+pub fn cargo_expand(path: &str, module: Option<&str>) -> String {
+    let mut cache = CACHE.lock().unwrap();
+    match cache.get(path) {
+        Some(expanded) => {
```

Comment:
nit: https://stackoverflow.com/questions/28512394/how-to-lookup-from-and-insert-into-a-hashmap-efficiently

Reply:
Thanks for the hint. Makes that function much simpler.